app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+class User < ActiveRecord::Base
+
+  has_many :pictures
+  has_many :comments
+  has_many :likes
+
+  devise :omniauthable, :omniauth_providers => [:facebook]
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)